test/check-machines-create

```diff
@@ -31,7 +31,7 @@ sys.path.append(os.path.join(TEST_DIR, "common"))
 sys.path.append(os.path.join(os.path.dirname(TEST_DIR), "bots/machine"))
 
 from machineslib import VirtualMachinesCase  # noqa
-from testlib import nondestructive, skipImage, test_main, wait, Error  # noqa
+from testlib import nondestructive, skipImage, test_main, wait, Error # noqa
```

Comment:
Is there any reason for this, or just a typo?
Also I remember we had a E261 rule, but I guess we dropped it.

Reply:
Argh, fallout from temporarily adding "sit" there. Fixed.

The static code tests are indeed broken, I fixed them in PR #788.

Reply:
I don't think that fixed this specific static-code check. I tried with the latest main checkout, and make codecheck will not catch the E261.

Reply:
@skobyda : Do you have "flake8" installed? That's what I tried to make more prominent with commit c68e54886315. 

Now that the fix landed, I'll continue on https://github.com/cockpit-project/cockpituous/pull/517 so that flake8 will actually run in CI.

Reply:
I do. I have problem with eslint and python3-vulture, but flake8 works for me just fine, and mentiones ntohign about E261:
```
[skobyda@fedora machines]$ make codecheck
test/static-code --tap
1..8
not ok 1 /static-code/test-eslint
# 
# WARNING (@babel/preset-env): We noticed you're using the `useBuiltIns` option without declaring a core-js version. Currently, we assume version 2.x when no version is passed. Since this default version will likely change in future versions of Babel, we recommend explicitly setting the core-js version you are using via the `corejs` option.
# 
# You should also be sure that the version you pass to the `corejs` option matches the version specified in your `package.json`'s `dependencies` section. If it doesn't, you need to run one of the following commands:
# 
#   npm install --save core-js@2    npm install --save core-js@3
#   yarn add core-js@2              yarn add core-js@3
# 
# More info about useBuiltIns: https://babeljs.io/docs/en/babel-preset-env#usebuiltins
# More info about core-js: https://babeljs.io/docs/en/babel-preset-env#corejs
ok 2 /static-code/test-flake8
ok 3 /static-code/test-include-config-h
ok 4 /static-code/test-js-translatable-strings
ok 5 /static-code/test-json-verify
ok 6 /static-code/test-no-translatable-attr
ok 7 /static-code/test-pyvulture # SKIP no python3-vulture
ok 8 /static-code/test-unsafe-security-policy

```